build.gradle

```diff
@@ -6,16 +6,33 @@ apply plugin: 'com.gradle.build-scan'
 
 // Adds support for Node.js scripts
 apply plugin: 'com.moowork.node'
+apply plugin: 'hsql-plugin'
+repositories {
+    // Use jcenter for resolving your dependencies.
+    // You can declare any Maven/Ivy/file repository here.
+    jcenter()
+    mavenCentral()
+    mavenLocal()
+}
+
+dbSettings {
+	database= "file:./uPortal/data/uPortal;hsqldb.tx=mvcc";
+	dbName = "uPortal";
+	port= "8887";
+
+}
 
 buildscript {
     repositories {
         maven {
             url 'https://plugins.gradle.org/m2/'
         }
+        mavenLocal()
     }
     dependencies {
         classpath 'com.gradle:build-scan-plugin:1.0'
         classpath "com.moowork.gradle:gradle-node-plugin:0.13"
+        classpath 'com.uportal.hsql:uportal-hsql:5.0.0-SNAPSHOT'
```

Comment:
When you execute ant hsql on local, you will see following line 
`Installing /home/stalele/uportal/uPortal/bootstrap/hsqldb/target/hsqldb-5.0.0-SNAPSHOT.jar to /home/stalele/.m2/repository/org/jasig/portal/hsqldb/5.0.0-SNAPSHOT/hsqldb-5.0.0-SNAPSHOT.jar
`
Thats the reason I also gave the same snapshot number